brands/amenity/cafe.json

```diff
@@ -35,6 +35,24 @@
       "takeaway": "yes"
     }
   },
+  "amenity/cafe|Aroma Espresso Bar~(International)": {
```

Comment:
You can leave off the International bit here

Reply:
I just merged it and fixed it in 124d64d. Thanks @mparrault :+1: